Xamarin.Forms.Core.UnitTests/ButtonUnitTest.cs

```diff
@@ -323,5 +324,29 @@ private void AssertButtonContentLayoutsEqual(Button.ButtonContentLayout layout1,
 			Assert.AreEqual(layout1.Position, bcl.Position);
 			Assert.AreEqual(layout1.Spacing, bcl.Spacing);
 		}
+
+		[Test]
+		public void ButtonCornerRadiusSetInExplicitStyle()
```

Comment:
should be moved to Xamarin.Forms.Xaml.UnitTests if the same behavior can't reproduced in a code-only test

Reply:
but ideally you should reproduce this test in code only, and leave it there

Reply:
I tried to reproduce it in code, but i didn't manage to reproduce the bug because its kinda dependent on the styles being applied and unapplied in the right order as the xaml is doing it.
But I moved it to Xamarin.Forms.Xaml.UnitTests as you wanted.